### ISSUE TITLE
Warnings op de P1-dongle-pro

### ISSUE THREAD

**Issue by bsjouke** [mhendriks/esphome-p1#4]
Ik zie dat er nu een constante stroom warnings wordt gegenereerd op de dongle:

16:09:41 | [W] | [component:205] | Components should block for at most 20-30ms.
16:09:42 | [W] | [component:204] | Component dsmr took a long time for an operation (0.13 s).
16:09:42 | [W] | [component:204] | Component dsmr took a long time for an operation (0.13 s).
16:09:42 | [W] | [component:205] | Components should block for at most 20-30ms.
16:09:43 | [W] | [component:204] | Component dsmr took a long time for an operation (0.10 s).
16:09:43 | [W] | [component:205] | Components should block for at most 20-30ms.
16:09:44 | [W] | [component:204] | Component dsmr took a long time for an operation (0.11 s).
16:09:44 | [W] | [component:205] | Components should block for at most 20-30ms.
16:09:45 | [W] | [component:204] | Component dsmr took a long time for an operation (0.10 s).
16:09:45 | [W] | [component:205] | Components should block for at most 20-30ms.
16:09:46 | [W] | [component:204] | Component dsmr took a long time for an operation (0.10 s).
16:09:46 | [W] | [component:205] | Components should block for at most 20-30ms.
16:09:47 | [W] | [component:204] | Component dsmr took a long time for an operation (0.10 s).
16:09:47 | [W] | [component:205] | Components should block for at most 20-30ms.
16:09:48 | [W] | [component:204] | Component dsmr took a long time for an operation (0.12 s).
16:09:48 | [W] | [component:205] | Components should block for at most 20-30ms.
16:09:49 | [W] | [component:204] | Component dsmr took a long time for an operation (0.11 s).
16:09:49 | [W] | [component:205] | Components should block for at most 20-30ms.
16:09:50 | [W] | [component:204] |  

etc etc.

Geen idee wanneer dit is gestart. ESP Home versie 2023.8.1

Het zijn warnings en de sensordata komt gewoon door. 

Enig idee?

**Comment by mhendriks** [comment]
Veel gebruikers van diverse ESPHome componenten hebben hier last van. zie https://github.com/esphome/issues/issues/4717

Denk dat dit een nieuwe "feature" is .

**Comment by Robby-Swartenbroekx** [comment]
Ja, ik heb dit ook al eventjes gezien. Al enkele maanden zelfs. Naar mijn gevoel van in april of mei. Had er nog niets van gezegd omdat het wel perfect blijft werken.
